cpp/test/prims/sparse/distance.cu

```diff
@@ -175,9 +183,539 @@ const std::vector<SparseDistanceInputs<int, float>> inputs_i32_f = {
    {1.0f, 2.0f, 1.0f, 2.0f, 1.0f, 2.0f, 1.0f, 2.0f},
    {5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0,
     5.0},
-   raft::distance::DistanceType::InnerProduct
+   raft::distance::DistanceType::InnerProduct},
```

Comment:
Do you have tests for hellinger and jaccard?

Reply:
Tagged you in the test for Jaccard. I need to generate a pairwise hellinger to compare. Scikit-learn doesn't have it. I can do this in a separate PR if that's okay with you.